GitExtensions/Project.Loc.targets

```diff
@@ -42,7 +42,7 @@
     <!-- 2. Run TranslationApp.exe to update English*.xlf -->
     <Exec Command="$(_TranslationCommand)"
           LogStandardErrorAsError="false"
-          WorkingDirectory="$(GitExtensionsOutputPath)"
+          WorkingDirectory="$(_GitExtensionsOutputDir)"
```

Comment:
This seems to make no difference when the `TranslationApp` is started from Visual Studio.
I do _not_ see a need to make this work because:

I found that `build.cmd -loc` does everything necessary well.
So you could mention this command in `CONTRIBUTING.md`.

Reply:
> This seems to make no difference when the `TranslationApp` is started from Visual Studio.

Ah! `TranslationApp` must be run from the `GitExtensions` folder where `Translation` folder it.
To run it from VS try setting the working directory and pass "update" parameter.

These targets aren't invoked from VS, so this change won't be visible. It is visible from the cli though.

Reply:
> To run it from VS try setting the working directory and pass "update" parameter.

First: There is an easy solution, which just needs to be documented. It is not necessary to run the `TranslationApp` in VS.

Second: Neither nor. `TranslationApp` does not use command line arguments. The working directory does not matter at all:
```
        public static string GetTranslationDir()
        {
            return Path.Combine(Path.GetDirectoryName(typeof(Translator).Assembly.Location), "Translation");
        }
```

Reply:
> First: There is an easy solution, which just needs to be documented. It is not necessary to run the `TranslationApp` in VS.

Correct.
`TranslationApp` is a really outdated artifact, which we are stuck with (until we can refactor it out, it is very complicated though). 

> Second: Neither nor. `TranslationApp` does not use command line arguments.

Huh, indeed! Those were removed in https://github.com/gitextensions/gitextensions/pull/6840 by some RussKie fellow...

> The working directory does not matter at all:

You're likely correct, but it just looks incorrect to start the app from the wrong folder.